login.html

```diff
@@ -60,6 +66,27 @@
         <img src="./assets/images/blob.png" id="blob">
     </div>
 
+    <!-- FOOTER -->
+    <div class="footer">
+        <div class="row mx-0">
+          <div class="col-md-6 mx-auto">
+            <div class="row mx-0 align-items-center">
+              <div class="col-sm-6 d-flex my-3 py-auto flex-column align-items-center order-sm-1 order-2">
+                <img src="./assets/images/logo.png" class='navLogo ' alt="..">
+              </div>
+              <div class="col-sm-6 d-flex my-3 py-auto flex-column align-items-center justify-content-center
+                           footer-content order-md-2 order-1">
+                <h4>Check Out</h4>
+                <ul class='list-unstyled'>
+                  <li><a href="https://github.com/devscollab/skill-board" target="_blank"><i class="fab fa-github fa-2x footer-icon"></i></a></li>
+                  <li><a href="https://www.instagram.com/devscollab/?hl=en" target="_blank"><i class="fab fa-instagram fa-2x footer-icon"></i></a></li>
+                </ul>
```

Comment:
This list can be made horizontal by using display: flex and flex-direction: row

Reply:
Okay, I will do the changes